lib/chunks.js

```diff
@@ -428,8 +436,11 @@ module.exports.getRuntimeDirectoryForCourse = (course) => {
  * @param {import('../sync/course-db').CourseData} options.courseData
  */
 module.exports.createChunksSymlinks = async ({ coursePath, courseId, courseData }) =>  {
+    /* eslint-disable no-unreachable */
```

Comment:
Are we still thinking of doing symlinks on local dev?  The error is getting thrown for me on sync so we could probably remove this until it's implemented.

Reply:
Good catch! I guess I never actually tested local sync while a `chunksGenerator` 😄 -- just to make sure I understand, this is only a problem on local dev with `chunksGenerator` set to `true`, right?

And good question about the local dev plan. Doing some kind of symlinking to make local dev look more like prod is a good idea. At the moment `createChunksSymlinks()` unconditionally symlinks everything, so I worry that things like reading a file from another question will still work. I was wondering whether we should try and set up symlinks on the fly for chunks, so that when you access a question you get a unique temporary "course directory" that only has the minimal set of symlinks (the specific question, serverFilesCourse, course elements). This would involve:

* [ ] Change `ensureChunksForCourse()` to return the newly-created `coursePath` and eliminate `getRuntimeDirectoryForCourse()`.
* [ ] In `ensureChunksForCourse()`, if we are not a `chunksConsumer` then make a new temporary `coursePath` and put symlinks inside it for the list of `chunks` passed as an argument.

What do you think?

Reply:
I'm on board with the idea of having questions in separate "course" directories like this.  I'm not sure why we should be symlinking only if we are not a consumer?  Maybe we should be more explicit and check if we're generating chunks and not running in ec2?

At the moment, I'm okay with spinning this out to a separate issue (since I wouldn't consider it pressing atm) and we can just comment out this call or something.

Reply:
After discussion with @nicknytko we decided to just delete `createChunksSymlinks()` completely. For local dev (where this was used) we can run in one of two setups:
1. (Default) Both `chunksGenerator` and `chunksConsumer` are false. This uses course content straight out of the on-disk course directly, just like we've always done.
2. Set both `chunksGenerator` and `chunksConsumer` to true. This will push/pull chunks to the local mock S3 for testing, just like in production.